features/messages/requests/MarkAllReadButton.tsx

```diff
@@ -1,3 +1,4 @@
+import { makeStyles,Typography } from "@material-ui/core";
```

Comment:
Surprised Prettier hasn't reformatted this line to have a space between the comma. Also, it doesn't look like this file has actually changed so we could just undo this from being changed at all

Reply:
Yeah think this page just hadn't had prettier run on it, which is why it's changed on this PR, strange with the comma, I maybe did something weird there! I can undo the changes on this page, but would it be better to just add the space and keep the formatted page?

Reply:
In this occasion, I'd say go for it since it's only one file!

But in general, I'd say please avoid re-formatting lots of files unrelated to your change as it adds noise to the PR and makes it longer to review, so might make the reviewing process less effective.

Reply:
Fair enough :)